Cas 82/enable 0 day turnarounds

froddd opened the pull request:
# Context

https://dsdmoj.atlassian.net/browse/CAS-82

# Changes in this PR

Updates error messages for turnaround errors, and add hint to ensure value entered is zero or more.

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

<!-- Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible. You can monitor deployment failures in CircleCI
itself and application errors are found in
[Sentry](https://ministryofjustice.sentry.io/issues/?project=4504129156218880&referrer=sidebar&statsPeriod=24h).
Both events should be automatically sent to our [Slack
channel](https://mojdt.slack.com/archives/C048BJS7S2F). -->

Commit:
Update turnaround error messages

Commit:
Show correct error for negative numbers

Commit:
Update turnaround hint

froddd commented on server/utils/formUtils.ts:
```diff
@@ -196,3 +196,13 @@ export const parseNaturalNumber = (input: string): number => {
+    return Number.NaN
+  }
+  return Number.parseInt(input, 10)
+}
```
I'm aware that adding this when there is a `parseNaturalNumber` method just above is odd, but I'd be a tad worried that updating the existing method may introduce bugs. Open to suggestions -- the existing one could take an extra argument (`options`?), but would need to be renamed across the project anyway (`parseNumber`?)!

aliuk2012 replied:
I agree it is a bit odd to have similar methods and to try play spot the diff with the regex. I like suggestion of a single `parseNumber` method. Perhaps rename the existing `parseNaturalNumber` to `parseNumber` and by default that  method would parse natural numbers. Once thats done extend the `parseNumber` to accept an optional argument which could be `options`. This should hopefully reduce the risk of introducing bugs.

This optional argument could be a json object for different types of numbers we might want to parse. Do we have any fields asking for monetary values? maybe something `options = { allow_negatives: true, allow_decimals: false }` . For the time being the only place where `options` would be set is the turnaround times.

Commit:
Refactor parseNumber utility

This now takes an options argument to enable parsing negative and
decimal number strings. The method name is updated to reflect the
broader scope.

aliuk2012 reviewed (approved):
Awesome refactor much better to extend the existing parseNumber method 👏
